spring-xd-codec/src/main/java/org/springframework/xd/dirt/integration/bus/serializer/kryo/AbstractKryoCodec.java

```diff
@@ -38,17 +38,6 @@
  */
 public abstract class AbstractKryoCodec<T> extends AbstractCodec<T> {
```

Comment:
AbstractKryoCodec overrides deserialize(byte[] bytes) with the same exact implementation as the base class, will remove on merge.